jp/content.json

```diff
@@ -7038,24 +7038,29 @@
                 "name":"panel1",
                 "loadingAnimtnData1_colo1_title": "Animation",
                 "loadingAnimtnData1_colo2":[
-                  {"desc":["","150*15px/dp","Lottie animation"]}]
+                  {"desc":["","150*15px/dp","<a href='https://lottiefiles.com/share/JHROo3' target='_blank'>Lottie animation</a>"]}
```

Comment:
Why is this pointing to lottiefiles?

Reply:
@shreyanshp yes we got approval to remove those link and i have updated.(confired by sunder and Yagi san)

Reply:
Kindly approve and merge this PR.